lib/modulr/api/transfers_service.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Modulr
+  module API
+    class TransfersService < Service
+      def create( # rubocop:disable Metrics/ParameterLists
+        account_id:,
+        currency:,
+        amount:,
+        destination_type:,
+        destination_iban:,
+        destination_name:,
```

Comment:
destination fields should be received as a VO? In that case I find better to receive base fields instead a composed object, what do you think @devengoapp/dev ?

Reply:
As an API consumer, I prefer to deal with plain hashes:

```rb
Whatever::API.create({
  one: 'XYZ',
  another: 'HEY',
  destination: {
    just_another: { and_another: true },
    but_more: { and_more: true },
  },
})
```
It's good and noble we enforce as much strictness as we want in our final requests to the APIs, but I don't want to transfer that strictness and complexity to the consumer and the creation of its client code.

Reply:
Done